hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -318,7 +369,8 @@ private StorageDescriptor storageDescriptor(TableMetadata metadata, boolean hive
     return storageDescriptor;
   }
 
-  private long acquireLock() throws UnknownHostException, TException, InterruptedException {
+  // Visible for testing
+  protected long acquireLock() throws UnknownHostException, TException, InterruptedException {
```

Comment:
Required for injecting a lock release and second commit, we need to capture the args to this call to get the lock id